src/Providers/ReviewsIo/README.md

```diff
@@ -0,0 +1,53 @@
+# ReviewsIO
```

Comment:
I just realized that we're missing explanation of required reviewsIo config values in the readme -- would be nice to add either here or in that example payload where you show trustpilot example in the main readme

Reply:
I know you can find them in config file but it will help

![image](https://user-images.githubusercontent.com/58700660/95445795-5357cd80-0957-11eb-8d94-deffe59e1459.png)